docs/source/checks/nlp/data_integrity/plot_special_characters.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+"""
+.. _nlp__special_characters:
+
+Special Characters
+******************
+
+This notebook provides an overview for using and understanding the special characters check.
+
+**Structure:**
+
+* `Why check for special characters? <#why-check-for-text-data-duplicates>`__
+* `Generate data & model <#generate-data-model>`__
+* `Run the Check <#run-the-check>`__
+* `Define a Condition <#define-a-condition>`__
+
+Why check for special characters?
+===================================
+
+The ``SpecialCharacters`` check looks for text sample in which the percentage of special characters
+out of all characters is significant. Such samples can be an indicator for a problem in the data pipeline that
+require attention. Additionally, such examples may be problematic for the model to predict on.
+For example, a text sample with many emojis may be hard to
+predict on and a common mythology will be to replace them with a textual representation of the emoji.
```

Comment:
Would replace with an example of html elements, for example, that may have made their way into our data.
If we're keeping the emoiji example, not sure this is the comon methodology to replace emojies with text. You have an example.
At the bare minumum - mythology -> methodolgy :P

Reply:
This check is not important enough to add a new dataset example